net/kadnode/files/kadnode.init

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh /etc/rc.common
+
+START=95
+
+LIST_SEP="
+"
+KADNODE_BIN=/usr/bin/kadnode
+
+
+append_opts() {
+	local p vl v s="$1"; shift
+	for p in $*; do
+		config_get vl "$s" "$p"
+		local IFS_OLD="$IFS"; IFS="
+"
+		for v in $vl; do
+			[ -n "$v" ] && OPTS="$OPTS
+--${p//_/-} ${v//'/\\'}"
+		done
+		IFS="$IFS_OLD"
+	done
+}
+
+append_opts_boolean() {
+  local p v s="$1"; shift
+  for p in $*; do
+        config_get_bool v "$s" "$p"  0
+        [ $v -gt 0  ] && OPTS="$OPTS
+--${p//_/-}"
+  done
+}
+
+section_enabled() {
+	config_get_bool enabled "$1" 'enabled' 0
+	[ $enabled -gt 0 ]
+}
```

Comment:
you use this once.  fine, whatever, your style :)

Reply:
Yes, it's debatable. I prefer to make the code more expressive this way.